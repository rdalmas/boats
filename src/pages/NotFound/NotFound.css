*, :before, :after {
  box-sizing: border-box;
}

.not-found {
  background: url(../../img/lost.jpg) no-repeat bottom center;
  height: 100vh;
  width: 100vw;
  display: flex;
  align-items: center;
  justify-content: center;
  padding: 0;
}

.not-found-panel {
  position: relative;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100%;
  width: 100%;
  background-color: rgba(0, 0, 0, 0.5);
}

.title {
  margin: 0;
  font-size: 6em;
  font-weight: 100;
  color: white;
}

.text {
  font-size: 1.5em;
  line-height: 1.4;
  color: white;
}
