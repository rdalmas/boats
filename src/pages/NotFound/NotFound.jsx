import React from 'react';
import { Link } from "react-router-dom";

import Wrapper from "../../styled-components/Wrapper";
import Panel from '../../styled-components/Panel';
import lostImg from "../../img/lost.jpg";

export const NotFound = () => (
  <Wrapper background={lostImg}>
    <Panel>
      <div className="col-md-12 col-sm-8">
        <h1 className="row no-gutters white-text justify-content-center">Ops...</h1>
        <h5 className="white-text row no-gutters text justify-content-center">It seems that you're lost in the sea.</h5>
        <h5 className="white-text row no-gutters text justify-content-center">Let us help guide you out and get you back home.</h5>
        <div className="row no-gutters justify-content-center">
          <Link className="btn btn-secondary btn-lg" to={{pathname: "/"}} role="button">Home</Link>
        </div>
      </div>
    </Panel>
  </Wrapper>
);

export default NotFound;
