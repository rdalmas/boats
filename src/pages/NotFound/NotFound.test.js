import React from 'react';
import { shallow } from "enzyme";

import { NotFound } from './NotFound';

describe("Page - Not Found", () => {
  let component;
  beforeEach(() => {
    component = shallow(<NotFound />);
  });
  it('should render the Not Found Page correctly', () => {   
    expect(component).toMatchSnapshot();
  });
});