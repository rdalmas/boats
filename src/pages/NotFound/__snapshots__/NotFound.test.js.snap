// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page - Not Found should render the Not Found Page correctly 1`] = `
<styled.div
  background="lost.jpg"
>
  <styled.div>
    <div
      className="col-md-12 col-sm-8"
    >
      <h1
        className="row no-gutters white-text justify-content-center"
      >
        Ops...
      </h1>
      <h5
        className="white-text row no-gutters text justify-content-center"
      >
        It seems that you're lost in the sea.
      </h5>
      <h5
        className="white-text row no-gutters text justify-content-center"
      >
        Let us help guide you out and get you back home.
      </h5>
      <div
        className="row no-gutters justify-content-center"
      >
        <Link
          className="btn btn-secondary btn-lg"
          role="button"
          to={
            Object {
              "pathname": "/",
            }
          }
        >
          Home
        </Link>
      </div>
    </div>
  </styled.div>
</styled.div>
`;
